src/cdk/table/table.ts

```diff
@@ -351,6 +351,8 @@ export class CdkTable<T> implements AfterContentChecked, CollectionViewer, OnDes
   /**
    * Stream containing the latest information on what rows are being displayed on screen.
    * Can be used by the data source to as a heuristic of what data should be provided.
+   * 
+   * Note: this feature has not yet been implemented.
```

Comment:
Rather than saying that it isn't implemented, we should exclude it from the docs instead. You can do so by adding a `@docs-private` tag.